libs/dh/metering-point/feature-tabs/src/lib/processes-tab-content/processes-detail-item/dh-processes-detail-item.component.ts

```diff
@@ -24,7 +24,13 @@ import {
 import { ProcessDetail } from '@energinet-datahub/dh/shared/domain';
 import { DhSharedUiDateTimeModule } from '@energinet-datahub/dh/shared/ui-date-time';
 
-import { WattIconModule, WattIconSize } from '@energinet-datahub/watt';
+import {
+  WattIconModule,
+  WattIconSize,
+  WattValidationMessageModule,
+  WattValidationMessageType,
```

Comment:
issue: not 100% sure but this will probably result in lint error since `WattValidationMessageType` is not used anymore.